### ISSUE TITLE
There is no smooth scrolling in documents when using the touchpad

### ISSUE THREAD

**Issue by SokolovskyMaxim** [ONLYOFFICE/DesktopEditors#1194]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (RPM package)

### Version information

7.3.0.184

### Expected Behavior

Scrolling is played in jerks

### Actual Behavior

The sheet scrolls smoothly. The way it is done in the application help portal

### Additional information

I am providing a screencast for clarity. When scrolling from the touchpad, the sheet scrolls jerkily, but if you scroll the scrollbar, the scrolling is smooth
The application's help tab already uses smooth scrolling from the touchpad. Why not transfer the same experience to working with documents?

[](https://user-images.githubusercontent.com/106545001/216587258-fe0dc5e5-6c81-46c3-b5d9-9fd6afc021cf.webm)

**Comment by maruacat** [comment]
Hello, @SokolovskyMaxim 
I can confirm - this is bug, issue Bug 61121 in our private issue tracker.
Thank you for report.

**Comment by Heath123** [comment]
It would solve a lot of issues if it could just use the browser’s native scrolling

**Comment by Mr-Blue-Sky-Candy** [comment]
I do not have this issue on version 7.4.1.36 (deb) on Archlinux. Maybe, it no longer is?

**Comment by the-hottest-bulb** [comment]
Any update on this issue yet? I am using Fedora and have the same issue.

**Comment by trey-a-12** [comment]
Yes, this is still very much the case, especially on macOS in the Document Writer.

https://github.com/user-attachments/assets/42d80386-e5f7-43b9-b75c-6685e399080e

**Comment by BZsombor** [comment]
Same for me, no smooth scrolling on macOS 26.5 (25F71), onlyoffice version: 9.4.0 (563-129) Apple Silicon.

**Comment by AlessandroTischer** [comment]
Same on windows. Scrolling with mouse wheel is fast and smooth, trackpad is choppy.

**Comment by DenisMayorko** [comment]
> Same for me, no smooth scrolling on macOS 26.5 (25F71), onlyoffice version: 9.4.0 (563-129) Apple Silicon.

Same for me MacOS 26.3.1, OnlyOffice version: 9.4.0
